integration-tests/kafka/src/test/java/io/quarkus/it/kafka/streams/KafkaStreamsTest.java

```diff
@@ -35,26 +31,36 @@
 @QuarkusTest
 public class KafkaStreamsTest {
 
-    private static Producer<Integer, String> createProducer() {
+    private static Producer<Integer, Customer> createProducer() {
+        Properties props = new Properties();
+        props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:19092");
+        props.put(ProducerConfig.CLIENT_ID_CONFIG, "streams-test-producer");
+        props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, IntegerSerializer.class.getName());
+        props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonbSerializer.class.getName());
```

Comment:
Typically, if the user wants to use the serializer but configure it in the `application.properties` file it would be:
`io.quarkus.kafka.client.serde.JsonbSerde.JsonbSerializer`

I would prefer: `io.quarkus.kafka.client.serde.JsonbSerializer`.